frontend_new/src/components/Res.js

```diff
@@ -4,6 +4,8 @@ import ListItemText from "@material-ui/core/ListItemText";
 import ListItem from "@material-ui/core/ListItem";
 import List from "@material-ui/core/List";
 import { useFetch } from "./Get";
+import SimpleCard from "./Cards.tsx";
+import Grow from '@material-ui/core/Grow';
 
 function Res(props) {
   const [res, resStatus] = useFetch(
```

Comment:
**[Required, Maintainability]** Inline AJAX calls are a maintainability nightmare. We should be using services to mask API calls behind methods. The component itself shouldn't care about where the data comes from, it should only need to know what to call to get it. The service it references is designed to handle those specifics as an abstraction. Mixing abstraction layers is not a good idea and leads to over-coupling logic.

An article on consuming REST APIs with services in React: [https://dzone.com/articles/consuming-rest-api-with-reactjs](https://dzone.com/articles/consuming-rest-api-with-reactjs)

Reply:
This is an issue with UIUX-30 (currently the UI main branch), not the PR. It shouldn't be addressed in this PR.